extensions/wikia/FlowTracking/scripts/createPageTracking.js

```diff
@@ -0,0 +1,41 @@
+define(
+	'ext.wikia.flowTracking.createPageTracking',
+	['wikia.flowTracking', 'wikia.querystring', 'mw', 'wikia.document', 'wikia.window'],
+	function (flowTrack, QueryString, mw, document, window) {
+		'use strict';
+
+		var namespaceId = mw.config.get('wgNamespaceNumber'),
+			articleId = mw.config.get('wgArticleId'),
+			title = mw.config.get('wgTitle');
+
+		function trackOnEditPageLoad(editor) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)